server/graphql/v2/object/Order.js

```diff
@@ -142,6 +145,21 @@ export const Order = new GraphQLObjectType({
           }
         },
       },
+      fromAccountMember: {
```

Comment:
This property name doesn't make much sense to me. I propose `membership` instead. 

To figure a good name, maybe you should add a `description` and try to explain what it's returning.